From e225e778ac1dfebfe1d49dc43f7d096d0ac8f23d Mon Sep 17 00:00:00 2001
From: Varsha Maira <vmaira@habana.ai>
Date: Wed, 16 Apr 2025 13:47:15 +0300
Subject: [PATCH] [SW-218933] g3: specs: move enum mem_acces_mode to specs

Currently, the `mem_access_mode` enum is defined within individual Zephyr
apps.

This patch moves the enum to a common location under the `specs` directory
to enable reuse across multiple apps and avoid definitions in multiple
apps.

Change-Id: I5610df1bbf7c92ab7e4345c0ef9ef34b93b630e2
Signed-off-by: Varsha Maira <vmaira@habana.ai>
---

diff --git a/hl_sec_if.h b/hl_sec_if.h
index 337fe48..eeb8078 100644
--- a/hl_sec_if.h
+++ b/hl_sec_if.h
@@ -229,4 +229,10 @@
 	HL_SEC_RR_SEL_PCIE = 4,
 };
 
+enum mem_access_mode {
+	ACCESS_RO,
+	ACCESS_WR,
+	ACCESS_RW,
+};
+
 #endif /* _HABANALABS_SEC_IF_H_ */
